aten/src/ATen/native/cpu/FillKernel.cpp

```diff
@@ -60,7 +60,7 @@ void fill_kernel(TensorIterator& iter, const Scalar& value_scalar) {
             [=]() -> scalar_t { return value; },
             [=]() { return Vectorized<scalar_t>(value); });
       }),
-      AT_EXPAND(AT_ALL_TYPES_AND_COMPLEX), kBool, AT_EXPAND(AT_BAREBONES_UNSIGNED_TYPES)
+      AT_EXPAND(AT_ALL_TYPES_AND_COMPLEX), kBool, AT_EXPAND(AT_FLOAT8_TYPES), AT_EXPAND(AT_BAREBONES_UNSIGNED_TYPES)
```

Comment:
is this still needed after https://github.com/pytorch/pytorch/pull/128744/files ?

Reply:
It is not.